product/src/main/java/com/yas/product/controller/attribute/ProductAttributeController.java

```diff
@@ -0,0 +1,92 @@
+package com.yas.product.controller.attribute;
```

Comment:
we don't need to add sub package for this. Please also help me move ProductAttributeGroupController to controller package